app/scripts/metamask-controller.js

```diff
@@ -5767,7 +5809,7 @@ export default class MetamaskController extends EventEmitter {
   }
 
   _notifyChainChange() {
-    if (this.preferencesController.getUseRequestQueue()) {
+    if (this.preferencesController?.getUseRequestQueue()) {
```

Comment:
Do we still need this change?

Maybe similar to [this case](https://github.com/MetaMask/metamask-extension/pull/23130#discussion_r1504540531), in that it's no longer needed

Reply:
Reverted here: https://github.com/MetaMask/metamask-extension/pull/23130/commits/9af40f307ef4dcb2536aee62a5aef6ebb3126f6e